CMakeLists.txt

```diff
@@ -77,11 +78,14 @@ add_subdirectory(runtime)
 if (NOT TPP_INSIDE_IREE)
   add_subdirectory(tools)
   add_subdirectory(test)
-  add_subdirectory(benchmarks)
 
   set(BENCHMARK_DIR "${PROJECT_SOURCE_DIR}/benchmarks")
-  add_custom_target(benchmarks ${BENCHMARK_DIR}/driver.py -v --build ${PROJECT_BINARY_DIR}
-                    DEPENDS bench_mlp bench_matmul tpp-opt tpp-run xsmm_dnn_mlp
+  set(CONFIG_DIR "${BENCHMARK_DIR}/config")
+  # Run a small set of benchmarks with small iterations on a stable machine (non-turbo) to track simple performance
+  # A separate CI job should track longer running and larger benchmarks elsewhere
+  add_custom_target(benchmarks ${BENCHMARK_DIR}/driver.py -v --build ${PROJECT_BINARY_DIR} -n 100
```

Comment:
I know it's for testing now but we want to keep it `-n 100` should probably go to config JSON files.